narratives/ncov_sit-rep_2020-03-27.md

```diff
@@ -0,0 +1,2734 @@
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 6-->
+# [European and U.S. outbreaks are intertwined](https://nextstrain.org/ncov/2020-03-27?c=region&d=tree,map&f_region=Europe,North%20America&label=clade:A2a&m=div&p=full&legend=open)
+Across the tree, we also see lots of comingling of European and American samples, indicating that the origins of the U.S. and European outbreaks are deeply intertwined.
+<br><br>
+Artificially, this may appear to flow primarily from Europe to the U.S., rather than bidirectionally; however, we expect that this is mostly an effect of sampling bias as we currently have more data from Europe.
+
+<!-- There is no right side text -->
+
+
+
+
+<!-- ############ NORTH AMERICA ############# -->
+<!-- This is left-side text 7-->
+# [Updates for North America](https://nextstrain.org/ncov/2020-03-27?d=tree,map&f_region=North%20America&p=grid&r=location&legend=closed)
+
+* [Executive summary](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=1).
+* [Background resources](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=2).     
+* [About this data](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=3).  
+* [Updates for Europe](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=4).  
+* [Updates for North America](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=7).  
+* [Updates for Central & South America](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=11).  
+* [Updates for Asia](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=13).  
+* [Updates for Africa](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=15).  
+* [Updates for Oceania](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=17).  
+* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=19).  
+* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=20).  
+
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 8-->
+# [Transmission across the U.S. is complex and not well-understood](https://nextstrain.org/ncov/2020-03-27?d=tree,map&f_division=Connecticut,Washington&label=clade:B1&m=div&p=full)
+
+Even within the U.S., transmission patterns are highly complex.
+<br><br>
+For example, recent samples from the Grubaugh lab in Connecticut are both ancestral to and intermingled with samples from Washington state.
+While more data is needed to draw firm conclusions, this suggests that there has been recent transmission between even very distant parts of the country.
+<br><br>
+This highlights that the transmission patterns across the U.S. are far more complex than is represented in the currently available data.
+Additional data will help us understand what is likely a deeply interconnected national epidemic playing out amidst highly heterogeneous responses across states.
```

Comment:
I really love this sentence! But it's language is not simple, and may be challenging for lay readers. I've tried to clarify, but likely did not succeed. 
```suggestion
Additional data will help us understand what is likely a deeply interconnected national epidemic playing out against the backdrop of highly heterogeneous responses from states.
```